lib/3llo/commands/card/assign.rb

```diff
@@ -0,0 +1,45 @@
+module Tr3llo
+  module Command
+    module Card
+      class AssignCommand
+        def initialize(card_id, board_id)
+          @card_id = card_id
+          @board_id = board_id
+        end
+
+        def execute
+          interface.print_frame do
+            @user_id = prompt_for_user_id!(@board_id)
+            assign_card
```

Comment:
I would prefer doing

```rb
user_id = prompt_for_user_id!(@board_id)
assign_card(user_id)
```

what do you think?

Reply:
Looks better. Do you want me to update?

Reply:
Yes please.